server/routes.js

```diff
@@ -0,0 +1,30 @@
+const controller = require('./controllers');
+const router = require('express').Router();
+
+// ---                     ---
+// --- Books API endpoints ---
+// ---                     ---
+router.get('/search', controller.books.searchBooks);
+router.get('/books', controller.books.personalBooks); // FIXME: conflict with react-router endpoint
+router.get('/popularBooks', controller.books.popularBooks);
+
+router.post('/books', controller.books.addBook); // FIXME: conflict with react-router endpoint
```

Comment:
React-router endpoint doesn't seem to have `'/books'` route, seeing a `'/mybooks'` route instead, can we double confirm?